python/pyspark/sql/tests/test_datasources.py

```diff
@@ -211,7 +211,7 @@ def test_checking_csv_header(self):
                 ]
             )
             df = (
-                self.spark.read.option("header", "true")
+                self.spark.read.option("header", True)
```

Comment:
This test is shared in both Spark Connect and Classic so should verify the test here.